pages/api/images/[searchTerm].js

```diff
@@ -0,0 +1,20 @@
+export default async function searchTerm(request, response) {
+  const { searchTerm } = request.query;
+
+  if (!searchTerm) {
+    return;
+  }
+
+  const API = process.env.NEXT_PUBLIC_IMAGE_API_KEY;
```

Comment:
please rename this key to IMAGE_API_KEY

Reply:
done! As for the reasons mentioned before i would leave weather fetching untouched :)